src/components/styles.js

```diff
@@ -0,0 +1,3 @@
+import '@mapbox/mbx-assembly/dist/assembly.css';
```

Comment:
If assembly.js is pulled in through a cdn can we do the same for these sources? I don't have a strong desire to require these in via npm.

Reply:
CDN would make things a bit simpler. I can def switch to that!

Reply:
Ok quick update: I imagine you'll still want to include Mapbox GL and Mapbox GL Draw from npm. Adding their CSS via CDN adds a bit of risk for getting out of sync down the line, so I left those two CSS imports, but moved them to `app.js` instead of keeping the separate `styles.js` file. If there's a more streamlined way to keep them in sync, happy to go a different way!